spec/classes/foreman_proxy__spec.rb

```diff
@@ -38,6 +38,8 @@
         end
       end
 
+      let(:system_image_root) { '/var/lib/foreman-proxy/tftp/system_images' }
```

Comment:
If it's static, we usually have the static value in every example.

Reply:
Done.